pipeline/relation_embedder/router/src/main/scala/weco/pipeline/router/RouterWorkerService.scala

```diff
@@ -43,7 +45,11 @@ class RouterWorkerService[MsgDestination](
       case None =>
         Success(List(work.transition[Denormalised](Relations.none)))
       case Some(CollectionPath(path, _)) =>
-        pathsMsgSender.send(path).map(_ => Nil)
+        work.sourceIdentifier.identifierType match {
+          case SierraSystemNumber =>
+            pathConcatenatorMsgSender.send(path).map(_ => Nil)
+          case _ => pathsMsgSender.send(path).map(_ => Nil)
+        }
```

Comment:
nit: how about introducing an intermediate variable to highlight the common parts:

```suggestion
val sender = work.sourceIdentifier.identifierType match {
  case SierraSystemNumber => pathConcatenatorMsgSender
  case _ => pathsMsgSender
}

sender.send(path).map(_ => Nil)
```